app/assets/stylesheets/component_guide/application.scss

```diff
@@ -2,12 +2,12 @@
 
 @import "govuk_publishing_components/components/helpers/govuk-frontend-settings";
 @import "govuk_publishing_components/components/helpers/markdown-typography";
-@import "govuk-frontend/settings/all";
-@import "govuk-frontend/tools/all";
-@import "govuk-frontend/helpers/all";
-@import "govuk-frontend/core/all";
+@import "govuk/settings/all";
+@import "govuk/tools/all";
+@import "govuk/helpers/all";
+@import "govuk/core/all";
 
-@include _govuk-font-face-nta;
+@include _govuk-font-face-gds-transport;
```

Comment:
As an aside, the font-face should be included automatically the first time it's used – you shouldn't have to do this. If that's not working for you, it'd be good to know, as we consider this a private mixin at the minute.

Reply:
This should've been removed after dropping the compatibility mode.

Reply:
Removed